src/main/java/eiteam/esteemedinnovation/commons/EsteemedInnovation.java

```diff
@@ -88,15 +88,17 @@
     public static final String MISC_SECTION = "section.Misc.name";
     public static final String STEAMTOOL_SECTION = "section.SteamTools.name";
 
+    //TODO: Fix sound registering
```

Comment:
What is the issue that needs fixing?

Reply:
`SoundEvent`s, like `Block`s and `Item`s, need to be registered on their own registry event.